scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/InProcessTaskExecutor.java

```diff
@@ -204,11 +207,15 @@ public TaskResultImpl execute(TaskContext taskContext, PrintStream output, Print
      * @throws Throwable
      */
     private Serializable execute(TaskContext taskContext, PrintStream output, PrintStream error,
-            ScriptHandler scriptHandler, Map<String, String> thirdPartyCredentials, VariablesMap variables)
-            throws Throwable {
+            ScriptHandler scriptHandler, Map<String, String> thirdPartyCredentials, VariablesMap variables,
+            Map<String, Serializable> jobResults) throws Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)